src/main/java/com/blackducksoftware/integration/jira/task/issue/handler/JiraIssueServiceWrapper.java

```diff
@@ -93,15 +88,14 @@ public JiraIssueServiceWrapper(final IssueService jiraIssueService, final IssueM
     public static JiraIssueServiceWrapper createIssueServiceWrapperFromJiraServices(final JiraServices jiraServices, final JiraUserContext jiraUserContext, final Gson gson, final Map<PluginField, CustomField> customFieldsMap) {
         return new JiraIssueServiceWrapper(
                  jiraServices.getIssueService()
-                ,jiraServices.getIssueManager()
                 ,jiraServices.getCommentManager()
                 ,jiraServices.getWorkflowManager()
                 ,jiraServices.createIssuePropertyWrapper()
                 ,new IssueFieldCopyMappingHandler(jiraServices, jiraUserContext, customFieldsMap)
                 ,jiraUserContext.getJiraAdminUser()
                 ,customFieldsMap
-                ,gson,
-                jiraServices.getWatcherManager());
+                ,gson
+                ,jiraServices.getWatcherManager());
```

Comment:
do we want the commas on the left or right?

Reply:
@ekerwin and I talked a while back about readability in constructors/factory initializers with lots of parameters. We determined this was a good solution.